docs/development/flashing_debugging.md

```diff
@@ -28,7 +28,9 @@ For development with STM32 microcontrollers, usually the ST-Link in-circuit debu
 
 The Libre Solar boards usually contain a 5-pin or 6-pin header with the same pin-out as the SWD connector on the ST-Link/V2 of the Nucleo boards. In addition to that, you can use the **serial interface** built-in to the Nucleo board to transfer debug information via an additional 2-pin connector.
 
-Before you can flash the software, the device needs to be powered, either via USB or the battery.
+::: Warning
```

Comment:
Doesn't seem to work (maybe needs to be lower-case?).
See here: https://learn.libre.solar/b/update-flashing-chapter/development/flashing_debugging.html#st-link

Reply:
of course lower case :/
Fixed it

Reply:
Great, thanks!